.github/workflows/windows.yml

```
@@ -257,6 +265,13 @@ jobs:
             tflags: 'skiprun'
       fail-fast: false
     steps:
+      - name: 'install stunnel'
+        if: ${{ matrix.tflags != 'skipall' && matrix.tflags != 'skiprun' }}
+        run: |
+          choco install -y stunnel --force
+          echo %PROGRAMFILES(X86)%\stunnel\bin>>%GITHUB_PATH%
+        shell: cmd
```

```suggestion
        run: choco install --yes --no-progress --limit-output --timeout 180 stunnel --force
```
Instead of modifying the global PATH, how about appending it in the test run step?

---

Updated